rdtools/__init__.py

```diff
@@ -10,6 +13,17 @@
 from rdtools.filtering import poa_filter
 from rdtools.filtering import tcell_filter
 from rdtools.filtering import clip_filter
+from rdtools.filtering import normalized_filter
+# from rdtools.soiling import soiling_srr
+# from rdtools.soiling import monthly_soiling_rates
+# from rdtools.soiling import annual_soiling_ratios
```

Comment:
Were these import comments intentionally left?

Reply:
Yes it was intentional. Our intent here was that not importing those by default would help signal their experimental nature.